### PR TITLE
Add `ignoreWhenEmpty` options to singleline/multiline-html-element-content-newline

### PR DESCRIPTION
**Pull request by armano2**
#682

### CODE REVIEW

File: docs/rules/multiline-html-element-content-newline.md [hunk 1]
```diff
@@ -65,11 +65,14 @@ This rule enforces a line break before and after the contents of a multiline ele
 ```json
 {
     "vue/multiline-html-element-content-newline": ["error", {
+        "ignoreWhenEmpty": true,
         "ignores": ["pre", "textarea"]
     }]
 }
 ```
 
+- `ignoreWhenEmpty` ... allows having contents in one line, when given element has no content.
```
Comment (armano2) [inline docs/rules/multiline-html-element-content-newline.md]: this message is wierd

Comment (michalsnik) [reply docs/rules/multiline-html-element-content-newline.md]: Perhaps `disables reporting when element has no content`?

File: lib/rules/multiline-html-element-content-newline.js [hunk 2]
```diff
@@ -108,6 +114,12 @@ module.exports = {
         }
 
         const getTokenOption = { includeComments: true, filter: (token) => token.type !== 'HTMLWhitespace' }
+        if (ignoreWhenEmpty && node.children.length === 0) {
```
Comment (michalsnik) [inline lib/rules/multiline-html-element-content-newline.js]: What do you think about using single `if` statement?
```js
if (
  ignoreWhenEmpty &&
  node.children.length === 0 &&
  template.getFirstTokensBetween(node.startTag, node.endTag, getTokenOption).length === 0
) {
  return
}

Comment (armano2) [reply lib/rules/multiline-html-element-content-newline.js]: i wanted to make it clean but with this wrapping its looks ok 👍

**Pull request merged**